modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -470,43 +470,104 @@ definition {
 	test CanCreateEntryWithAggregationField {
 		property portal.acceptance = "true";
 
-		task ("Create Custom Object and Custom Relationship itself (via API)") {
+		task ("Given two object definitions related") {
```

Comment:
`Given two related object definitions`

Adjectives then nouns, it's the opposite in English.